src/Gizmos/gizmo.ts

```diff
@@ -161,7 +163,33 @@ export class Gizmo implements IDisposable {
         if (!this._attachedNode) {
             return;
         }
-        if (this._attachedNode.getClassName() === "Mesh" || this._attachedNode.getClassName() === "AbstractMesh" || this._attachedNode.getClassName() === "TransformNode") {
+        var inheritsTargetCamera = this._attachedNode.getClassName() === "FreeCamera"

```

Comment:
Just a thought - I am not sure how often this function is executed, but - if it is executed every frame, maybe first check if classname contains camera, to avoid all of those comparisons (and skip in case of a mesh)

Reply:
On the `Camera` class you have a hidden public read only property `_isCamera = true`: maybe just test for it?